deploy/test/config/k8s_secrets.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ ID }}-test-k8s-secret
```

Comment:
i am guessing you want two k8s secrets to test a scenario in which we can retrieve them both. if so, i would create 2 different files for explicitness.

Reply:
i remember that there was a previous comment to join them into one (:

Reply:
well, i disagree with the previous comment

Reply:
imo I think it looks cleaner having a file that can be dynamically changed like this one for less overhead